Gems/Atom/RHI/DX12/Code/Source/RHI/CommandList.h

```diff
@@ -401,6 +402,36 @@ namespace AZ
             {
                 const size_t srgSlot = pipelineLayout->GetSlotByIndex(srgIndex);
                 const ShaderResourceGroup* shaderResourceGroup = bindings.m_srgsBySlot[srgSlot];
+                RootParameterBinding binding = pipelineLayout->GetRootParameterBindingByIndex(srgIndex);
+
+                if (binding.m_bindlessTable.IsValid())
```

Comment:
Add a quick comment about binding the global static heap for indirect views here.